### ISSUE TITLE
GE generating example not support ATHENA "NotImplementedError: AWS Athena does not support OFFSET"

### ISSUE THREAD

**Issue by tuannguyen0901** [fivetran/great_expectations#2032]
**Describe the bug**
NotImplementedError: AWS Athena does not support OFFSET.

**To Reproduce**
1. Following CLI "great_expectations init" process
2. Until "Which table would you like to use?" --> select * from tableXYZ
3. GE generating example ...
4. See error "NotImplementedError: AWS Athena does not support OFFSET."

**Expected behavior**
Unsupported features should be skipped while using "great_expectations init"

**Environment (please complete the following information):**
 - OS: Amazon Linux
 - GE Version: version 0.12.7

**Additional context**
Using Athena as data source

FULL ERROR MESSAGE: 

```
Would you like to profile new Expectations for a single data asset within your new Datasource? [Y/n]: y

Which table would you like to use? (Choose one)
    1. default.fact_ro (table)
    2. default.tbdealer_updated (table)
    3. default.tbdealer_updated_underlying_table (table)
    4. default.testwr (table)
    5. dfx_clean_mtproduction.tbalappointmentmetadata (table)
    Do not see the table in the list above? Just type the SQL query
: SELECT * FROM mart_product.fact_appointments_details
Created permanent table default.ge_tmp_8f08f7d5

Name the new Expectation Suite [query.warning]: QA_fact_appointments_detail 

Great Expectations will choose a couple of columns and generate expectations about them
to demonstrate some examples of assertions you can make about your data.

Great Expectations will store these expectations in a new Expectation Suite 'QA_fact_appointments_detail' here:

  file:///home/ec2-user/environment/ge_example/great_expectations/expectations/QA_fact_appointments_detail.json

Would you like to proceed? [Y/n]: y

Generating example Expectation Suite...
Created permanent table default.ge_tmp_1a831e5f

Traceback (most recent call last):
  File "/usr/local/bin/great_expectations", line 11, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.6/site-packages/great_expectations/cli/cli.py", line 72, in main
    cli()
  File "/usr/local/lib64/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib64/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib64/python3.6/site-packages/click/core.py", line 1259, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib64/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib64/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/great_expectations/cli/init.py", line 156, in init
    open_docs=False,
  File "/usr/local/lib/python3.6/site-packages/great_expectations/cli/toolkit.py", line 143, in create_expectation_suite
    profiler_configuration,
  File "/usr/local/lib/python3.6/site-packages/great_expectations/cli/toolkit.py", line 199, in _profile_to_create_a_suite
    additional_batch_kwargs=additional_batch_kwargs,
  File "/usr/local/lib/python3.6/site-packages/great_expectations/data_context/data_context.py", line 2045, in profile_data_asset
    batch, run_id=run_id, profiler_configuration=profiler_configuration
  File "/usr/local/lib/python3.6/site-packages/great_expectations/profile/base.py", line 225, in profile
    data_asset, configuration=profiler_configuration
  File "/usr/local/lib/python3.6/site-packages/great_expectations/profile/basic_suite_builder_profiler.py", line 408, in _profile
    return cls._demo_profile(dataset)
  File "/usr/local/lib/python3.6/site-packages/great_expectations/profile/basic_suite_builder_profiler.py", line 551, in _demo_profile
    cls._create_expectations_for_numeric_column(dataset, column)
  File "/usr/local/lib/python3.6/site-packages/great_expectations/profile/basic_suite_builder_profiler.py", line 213, in _create_expectations_for_numeric_column
    column, min_value=None, max_value=None, result_format="SUMMARY"
  File "/usr/local/lib/python3.6/site-packages/great_expectations/data_asset/util.py", line 80, in f
    return self.mthd(obj, *args, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/great_expectations/data_asset/data_asset.py", line 275, in wrapper
    raise err
  File "/usr/local/lib/python3.6/site-packages/great_expectations/data_asset/data_asset.py", line 260, in wrapper
    return_obj = func(self, **evaluation_args)
  File "/usr/local/lib/python3.6/site-packages/great_expectations/dataset/dataset.py", line 134, in inner_wrapper
    evaluation_result = func(self, *args, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/great_expectations/dataset/dataset.py", line 2755, in expect_column_median_to_be_between
    column_median = self.get_column_median(column)
  File "/usr/local/lib/python3.6/site-packages/great_expectations/dataset/sqlalchemy_dataset.py", line 760, in get_column_median
    raise NotImplementedError("AWS Athena does not support OFFSET.")

```

**Comment by eugmandel** [comment]
Thank you for reporting this, @tuannguyen0901 ! I wish all database engines behaved in the same way, but this is only a wish :). From time to time we discover inconsistencies like this one. The solution is usually to implement a special case for that database. In this case it looks like we need an Athena-specific implementation of get_column_median.

**Comment by kuhnen** [comment]
@eugmandel  I am just starting to use this great tool :). Could you please point wherein the code we could try to implement the athena specific case?

**Comment by kuhnen** [comment]
@eugmandel @tuannguyen0901  I believe this issue was solved on version  0.13.13 and an additional feature on 0.13.18.

https://github.com/great-expectations/great_expectations/pull/2544
https://github.com/great-expectations/great_expectations/pull/2521

Maybe we should mark this as solved?
